edualert/statistics/tasks.py

```diff
@@ -167,3 +176,246 @@ def get_months_since_academic_calendar_start(current_calendar):
+        worksheet.column_dimensions['B'].width = max_width - worksheet.column_dimensions['A'].width
+
+        worksheet.column_dimensions['D'].width = len(worksheet['D12'].value)
+        worksheet.column_dimensions['E'].width = len(worksheet['E12'].value)
+        worksheet.column_dimensions['F'].width = len(worksheet['F12'].value)
+
+        # begin writing table data
+        total_founded_absences = 0
+        total_unfounded_absences = 0
+        row = 13
+        study_width = worksheet.column_dimensions['B'].width + worksheet.column_dimensions['C'].width
+        for index, study in enumerate(clazz['subjects']):
+            study_name = study['subject_name']
+            founded_absences = study['founded_absences']
+            unfounded_absences = study['unfounded_absences']
+
+            # sum absences for Total row
+            total_founded_absences += founded_absences
+            total_unfounded_absences += unfounded_absences
+
+            # write subject row
+            worksheet.cell(column=1, row=row, value=index + 1)
+            cell = worksheet.cell(column=2, row=row, value=study_name)
+            cell.alignment = Alignment(wrap_text=True, vertical='justify', shrink_to_fit=True)
+            worksheet.merge_cells('B{}:C{}'.format(row, row))
+            worksheet.cell(column=4, row=row, value=founded_absences)
+            worksheet.cell(column=5, row=row, value=unfounded_absences)
+            worksheet.cell(column=6, row=row, value=founded_absences + unfounded_absences)
+
+            # adjust height if necessary
+            if len(str(cell.value)) > study_width:
+                height = math.ceil(len(cell.value) / study_width) * 12
+                worksheet.row_dimensions[row].height = height
+
+            # set the borders for the cells
+            for column in range(1, 7):
+                cell = worksheet.cell(column=column, row=row)
+                set_border(cell)
+            # move to next row
+            row += 1
+
+        # write Total row
+        cell = worksheet.cell(column=1, row=row, value='TOTAL')
+        cell.font = Font(name='Calibri', bold=True)
+        cell.alignment = Alignment(horizontal='center', vertical='center')
+        worksheet.merge_cells(start_row=row, start_column=1, end_row=row, end_column=3)
+        worksheet.cell(column=4, row=row, value=total_founded_absences)
+        worksheet.cell(column=5, row=row, value=total_unfounded_absences)
+        worksheet.cell(column=6, row=row, value=total_founded_absences + total_unfounded_absences)
+        # set the borders for the cells
+        for column in range(1, 7):
+            cell = worksheet.cell(column=column, row=row)
+            set_border(cell)
+
+    # add sheet to allow easier change for global values
+    worksheet = workbook.create_sheet('Cap')
+    worksheet['A1'] = 'Termen de predare:'
+    worksheet['B1'] = 'Setat din Cap:B1'
```

Comment:
this is should be  "Data predării:" and the above one is not used at all. Do we need to use them? I kinda feel like it's error prone.